src/FWlib/peripheral_pinctrl.h

```diff
@@ -514,6 +514,8 @@ int PINCTRL_SelPCAPIn(int index, uint8_t io_pin);
 int PINCTRL_SelQDECIn(uint8_t phase_a,
                       uint8_t phase_b);
 
+int PINCTRL_SelQDECPcapIn(int index, uint8_t io_pin);
+
```

Comment:
`QDECPcap` looks confusing.

Reply:
I have creat pwm timer and pacp operation interface for qdec module, here temporarily delete pinctrl's multiplexing interface, I will change it in another pr. Now other is repaired.